### PR TITLE
Allow CLI to limit enumerate search to specific device type

### PR DESCRIPTION
**Pull request by PierreRochard**
This is helpful when debugging a specific device module import/dependency issue.

### CODE REVIEW

**Comment by instagibbs** [comment]
argh travis is down, we've been flagged for "abuse". Probably because "mining"

**Comment by achow101** [comment]
Closing and reopening to trigger travis

File: hwilib/commands.py [hunk 1]
```diff
@@ -30,14 +31,22 @@ def get_client(device_type, device_path, password=''):
...
+        modules = all_devs
+    else:
+        modules = [device_type]
+
+    for module in modules:
         try:
             imported_dev = importlib.import_module('.devices.' + module, __package__)
             result.extend(imported_dev.enumerate(password))
-        except ImportError:
+        except ImportError as e:
```
Comment (achow101) [inline hwilib/commands.py]: This block is an unrelated change.

File: hwilib/cli.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def displayaddress_handler(args, client):
     return displayaddress(client, desc=args.desc, path=args.path, sh_wpkh=args.sh_wpkh, wpkh=args.wpkh)
 
 def enumerate_handler(args):
-    return enumerate(password=args.password)
+    return enumerate(password=args.password, device_type=args.device_type)
```
Comment (achow101) [inline hwilib/cli.py]: I'm not sure that this is safe if device type is not specified.

Also, the device type may not necessarily be just the type, it could also be the model name in which the changes to `enumerate()` would not work.

**Comment by instagibbs** [comment]
you failed to trigger travis apparently. @PierreRochard could you clean up PR and push again to see if it gets triggered?

**Comment by achow101** [comment]
Closing due to lack of activity. marking as up for grabs
